src/client/components/Avatar/Avatar.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Avatar = (props) => {
+  return (
+    <div className="avatar-images-container">
```

Comment:
What is the reason you need the `<div>` wrapping the `<img/>` ?

Reply:
I used th <div> to wrap the <img/> for styling purposes.  The <div> makes makes the image positioning easy, and Priyanka adviced me yesterday aswell that it is necessary if i will want to make a circle and a background color around the image.

Reply:
That makes sense 👍